nerium/contrib/formatter/affix.py

```diff
@@ -13,5 +13,6 @@ def format_results(self):
         formatted['response'] = self.result
         formatted['metadata'] = {}
         formatted['metadata']['executed'] = datetime.datetime.now().isoformat()
+        # TODO: Lib shouldn't depend on Flask. Get these params another way
```

Comment:
Maybe we could pass that kind of metadata through the resultset object...?

Reply:
Yeah, I was thinking about the same thing. We just need to pass the same kwargs to the formatter that we do through the resultset. "through the resultset" is one way, but right now the Flask app calls the resultset and then the formatter, so it might just repeat the call. (Implementation detail)